demo/src/html/index.html

```diff
@@ -3,22 +3,14 @@
     <head>
         <meta charset="UTF-8" />
         <title><%= htmlWebpackPlugin.options.title %></title>
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+        <link
+            href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap"
+            rel="stylesheet"
+        />
     </head>
     <body>
-        <header class="container">
-            <hgroup>
-                <h1><%= htmlWebpackPlugin.options.title %></h1>
-                <p>
-                    Demos of the
-                    <a href="https://github.com/david-yz-liu/memory-viz"
-                        >MemoryViz</a
-                    >
-                    Javascript library for visualizing Python memory.
-                </p>
-            </hgroup>
-        </header>
-        <main class="container">
```

Comment:
We should not lose the `main` element here; you can add this into the `App` component.

Reply:
Hi Professor, just to confirm that we're understanding you correctly, would you like us to wrap `App.tsx` in `<main className="container">`?

So the component looks like:
```
export default function App() {
      return (
          <main className="container">
              ...Code
          </main>
      )
}
```